docs/_static/wagtail_icons_table.txt

```diff
@@ -488,7 +489,7 @@
 <td> earth-americas (solid): Font Awesome Pro 6.4.0 </td>
 <td><code>wagtailadmin/icons/site.svg</code></td> </tr>
 <tr>
-<td><svg width="32" height="32" fill="currentColor"><svg id="icon-sliders" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><!--! sliders (solid): Font Awesome Pro 6.4.0 --><path d="M0 13C0 12.4688 0.4375 12 1 12H2.6875C3.09375 11.125 3.96875 10.5 5 10.5C6 10.5 6.875 11.125 7.28125 12H15C15.5312 12 16 12.4688 16 13C16 13.5625 15.5312 14 15 14H7.28125C6.875 14.9062 6 15.5 5 15.5C3.96875 15.5 3.09375 14.9062 2.6875 14H1C0.4375 14 0 13.5625 0 13ZM6 13C6 12.4688 5.53125 12 5 12C4.4375 12 4 12.4688 4 13C4 13.5625 4.4375 14 5 14C5.53125 14 6 13.5625 6 13ZM11 5.5C12 5.5 12.875 6.125 13.2812 7H15C15.5312 7 16 7.46875 16 8C16 8.5625 15.5312 9 15 9H13.2812C12.875 9.90625 12 10.5 11 10.5C9.96875 10.5 9.09375 9.90625 8.6875 9H1C0.4375 9 0 8.5625 0 8C0 7.46875 0.4375 7 1 7H8.6875C9.09375 6.125 9.96875 5.5 11 5.5ZM12 8C12 7.46875 11.5312 7 11 7C10.4375 7 10 7.46875 10 8C10 8.5625 10.4375 9 11 9C11.5312 9 12 8.5625 12 8ZM15 2C15.5312 2 16 2.46875 16 3C16 3.5625 15.5312 4 15 4H8.28125C7.875 4.90625 7 5.5 6 5.5C4.96875 5.5 4.09375 4.90625 3.6875 4H1C0.4375 4 0 3.5625 0 3C0 2.46875 0.4375 2 1 2H3.6875C4.09375 1.125 4.96875 0.5 6 0.5C7 0.5 7.875 1.125 8.28125 2H15ZM5 3C5 3.5625 5.4375 4 6 4C6.53125 4 7 3.5625 7 3C7 2.46875 6.53125 2 6 2C5.4375 2 5 2.46875 5 3Z" /></svg></td>
+<td><svg width="32" height="32" fill="currentColor"><svg id="icon-sliders" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><!--! sliders (solid): Font Awesome Pro 6.4.0 --><path d="M0 13C0 12.4688 0.4375 12 1 12H2.6875C3.09375 11.125 3.96875 10.5 5 10.5C6 10.5 6.875 11.125 7.28125 12H15C15.5312 12 16 12.4688 16 13C16 13.5625 15.5312 14 15 14H7.28125C6.875 14.9062 6 15.5 5 15.5C3.96875 15.5 3.09375 14.9062 2.6875 14H1C0.4375 14 0 13.5625 0 13ZM6 13C6 12.4688 5.53125 12 5 12C4.4375 12 4 12.4688 4 13C4 13.5625 4.4375 14 5 14C5.53125 14 6 13.5625 6 13ZM11 5.5C12 5.5 12.875 6.125 13.2812 7H15C15.5312 7 16 7.46875 16 8C16 8.5625 15.5312 9 15 9H13.2812C12.875 9.90625 12 10.5 11 10.5C9.96875 10.5 9.09375 9.90625 8.6875 9H1C0.4375 9 0 8.5625 0 8C0 7.46875 0.4375 7 1 7H8.6875C9.09375 6.125 9.96875 5.5 11 5.5ZM12 8C12 7.46875 11.5312 7 11 7C10.4375 7 10 7.46875 10 8C10 8.5625 10.4375 9 11 9C11.5312 9 12 8.5625 12 8ZM15 2C15.5312 2 16 2.46875 16 3C16 3.5625 15.5312 4 15 4H8.28125C7.875 4.90625 7 5.5 6 5.5C4.96875 5.5 4.09375 4.90625 3.6875 4H1C0.4375 4 0 3.5625 0 3C0 2.46875 0.4375 2 1 2H3.6875C4.09375 1.125 4.96875 0.5 6 0.5C7 0.5 7.875 1.125 8.28125 2H15ZM5 3C5 3.5625 5.4375 4 6 4C6.53125 4 7 3.5625 7 3C7 2.46875 6.53125 2 6 2C5.4375 2 5 2.46875 5 3Z" /></svg></svg></td>
```

Comment:
It seems we nest `<svg>` elements in this table, likely because the snippet in the Styleguide only gives you `<svg><use>` since we used sprites for our icons. I'm not sure of the exact process, but it looks like this table is now updated either manually, or with the JS snippet + manually replacing the `<use>` element with the actual icon's `<svg>`.

This case in particular was missing a closing `</svg>` tag, likely due to manual work involved.